app/scripts/dimApp.i18n.js

```diff
@@ -500,35 +500,171 @@
+            SupportDIM: "Soutien DIM"
+          },
+          Help: {
+            BackToDIM: 'Retour sur DIM',
+            Drag: 'Maintenez shift ou stoppez au-dessus d\'une zone de dépôt pour transférer une partie de la pile.'
+          },
+          LB: {
+            LB: "Constructeur de Loadout",
+            ShowGear: "Afficher équipement de {{class}}",
+            HideGear: "Cacher équipement de {{class}}",
+            LockEquipped: "Verrouiller équipé",
+            ClearLocked: "Enlever verrouillés",
+            Locked: "Articles verrouillés",
+            LockedHelp: 'Glissez et déplacez un article dans cet espace pour construire votre set avec cet équipement. Shift + clique pour exclure les articles',
+            FilterSets: "Filtrer les sets",
+            AdvancedOptions: "Options avancées",
+            ProcessingMode: {
+              Fast: "Rapide",
+              Full: "Complet",
+              ProcessingMode: "Mode de processing",
+              HelpFast: "Ne se concentre que sur votre meilleur équipement.",
+              HelpFull: "Se concentre sur plus d'équipements, mais prend plus de temps."
+            },
+            Scaled: "Échelonné",
+            Current: "Actuel",
+            LightMode: {
+              LightMode: "Mode lumière",
+              HelpScaled: 'Calcule les loadouts comme si tous les articles avaient 350 de défense.',
+              HelpCurrent: 'Calcule les loadouts à leur niveau de défense actuel.'
+            },
+            LBIncludeRare: 'Inclure les articles rares (bleus)',
+            Help: "Besoin d'aide?",
+            Equip: "Équiper sur le perso actuel",
+            ShowAllConfigs: "Afficher toutes les configurations",
+            ShowConfigs: "Afficher les configurations",
+            HideAllConfigs: "Cacher toutes les configurations",
+            HideConfigs: "Cacher les configurations",
+            Loading: "Chargement des meilleurs sets",
+            Vendor: "Inclure articles",
+            Exclude: "Articles exclus",
+            ExcludeHelp: "Shift + cliquez un article (ou glissez-déposez dans cet espace) pour construire des sets sans équipement spécifique.",
+            LockPerk: "Verrouiller amélioration",
+            Missing1: 'Pièces légendaires ou éxotiques manquantes pour construire un set complet!',
+            Missing2: 'Pièces légendaires ou éxotiques manquantes pour construire un set complet!'
+          },
           Loadouts: {
+            Loadouts: "Loadouts",
+            Before: 'Avant \'{{name}}\'',
             Create: "Créer Loadout",
             FromEquipped: "Equipé",
             Edit: "Modifier Loadout",
             Delete: "Effacer Loadout",
+            ConfirmDelete: "Êtes-vous sûr de vouloir supprimer '{{name}}'?",
             ApplySearch: "Articles = \"{{query}}\"",
             MaximizeLight: "Maximiser la Lumière",
             ItemLeveling: "Evolution d'article",
             GatherEngrams: "Rassembler les Engrammes",
```

Comment:
Can we shorten this any?  `Réunir` maybe
Same for `Farming Mode`. see attached
![screenshot 2016-11-11 11 45 27](https://cloud.githubusercontent.com/assets/4798491/20226217/95e0d60a-a804-11e6-9bf0-47740292f614.png)